src/util/instanceDuplicate.tsx

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "./queryKeys";
+import { fetchInstances } from "api/instances";
+import { LxdInstance } from "types/instance";
+
+export const getDuplicatedInstanceName = (instance: LxdInstance): string => {
+  const { data: instances = [] } = useQuery({
+    queryKey: [queryKeys.instances],
+    queryFn: () => fetchInstances(instance.project),
```

Comment:
```suggestion
    queryFn: () => fetchInstances(targetProject),
```
We should use the selected target project to pick a free name here, not the project the source instance lives in.

Reply:
I don't think this information is accessible at this time as this function is being called to pass an initialValue to the form, and also from within the form I do not believe it can access it's own values.

Reply:
You are right, the initial value of the target project will match the instance project. And if the user later changes the target project, we won't regenerate a name -- so it should be fine to use the instance project here.